master/reference/calicoctl/resources/policy.md

```diff
@@ -1,140 +0,0 @@
----
-title: Policy Resource (policy)
```

Comment:
If we are deleting this page, then we need to think about what we will do for the `latest` redirects. 
- If we can redirect people from the old `policy` to one of these new policy pages, great (ideal). I am not sure how these `latest` redirects get added and if that automation handles this case somehow (cc @ozdanborne ), but if we can redirect to one of these pages, LMK which one.
- If redirect people from the old `policy` to one of these new policy pages, then we need to add a `no-canonical: true` tag here to the metadata (not idea, but OK if necessary).

Reply:
OK, nvm, I think regardless of the `latest` redirect, we need to add the `no-canonical` tag to the pages that exist in previous directories. Otherwise, you will see the `htmlproofer` fail, I believe.